src/autoComplete/index.tsx

```diff
@@ -107,33 +109,44 @@ export default class AutoComplete extends Component<AutoCompleteProps, AutoCompl
     } else {
       this.searchForItems = this.searchForItems.bind(this)
```

Comment:
Wow, can you remove this please? 

No more bind this, use class properties instead

Reply:
done